Lib/swig.swg

```diff
@@ -673,7 +675,14 @@ template<typename T> class SwigValueWrapper {
 public:
   SwigValueWrapper() : pointer(0) { }
   SwigValueWrapper& operator=(const T& t) { SwigMovePointer tmp(new T(t)); pointer = tmp; return *this; }
-  operator T&() const { return *pointer.ptr; }
+  SwigValueWrapper& operator=(T&& t) {
+    if (pointer.ptr != &t) {
+      SwigMovePointer tmp(new T(std::move(t)));
+      pointer = tmp;
+    }
+    return *this;
+  }
+  operator const T&() const { return *pointer.ptr; }
```

Comment:
These changes produce generated code that requires a C++11 compiler.  I think you need to conditionalised the new code based on checking the value of `__cplusplus` so that it still works as currently for a pre-C++11 compiler.

I think your CI failures are due to CI testing that things work with `-std=c++98` (at least in part, and perhaps entirely).

Reply:
aha! Thanks.